### ISSUE TITLE
Perform your first parameter study

### ISSUE THREAD

**Issue by philippabele** [philippabele/deep-groove-with-pytorch#28]
Machine learning is known to be a computationally intensive process.

Based on your first data set and your first trained model, perform a first parameter study.

Decide on your own witch parameters to vary, but give an overview of the tested parameter space. in addition, think of important metrics in order to compare your different models.

Start simple and don't think too complicated, choose parameters that are easy to vary from a coding perspective.

It is more important to set up the first study than the actual variation. there will be many more in the future.

Divide your work into two parts. This issue is for coding. I am going to create a second one for documenting.

**Comment by RicoKursidem** [comment]
Hi, 
I'm not certain that I 100% understand what to do. 

I think I should vary some parameters and write down what happens. Like increasing the number of ReLu Layers, the number of nodes, learning rate, ...
It should produce an understanding of what every parameter does. Am I right?

EDIT: I implemented a study of learning rate, batch size and epochs as I thought I should do. If I need to change something or check more parameters, I will do that. 
I wrote a new script and made it possible to change some parameters from outside the cls_set1 function.
